pkg/chunk/chunk_test.go

```diff
@@ -196,7 +196,7 @@ func TestChunksToMatrix(t *testing.T) {
 
 func benchmarkChunk() Chunk {
 	// This is a real example from Kubernetes' embedded cAdvisor metrics, lightly obfuscated
-	return dummyChunkFor(model.Metric{
+	return dummyChunkFor(model.Now(), model.Metric{
```

Comment:
I suspect `now` should be a parameter here so the benchmark can be as consistent as it wants to be